include/groonga/groonga.h

```diff
@@ -1062,6 +1062,28 @@ grn_column_rename(grn_ctx *ctx,
                   const char *name,
                   unsigned int name_size);
 
+/**
+ * \brief Frees a temporary object from memory.
+ *
+ *        This function frees the specified object (`obj`) from memory.
+ *        If `obj` includes other associated objects, it recursively frees
+ *        those as well.
+ *
+ * \attention This function should not be used to free persistent objects like
+ *            tables, columns, or expr. These objects should be kept open for
+ *            efficiency, and \ref grn_obj_unlink should be used to manage them
+ *            appropriately instead of closing them.
+ *            Closing persistent objects repeatedly can lead to inefficiencies.
+ *            However, both persistent and temporary objects can be specified
+ *            with this function, but be mindful of the performance
+ *            implications.
```

Comment:
元のテキストが現状を反映していなかったのでもりっと変えてよさそう。（`expr`をpersistentなオブジェクトとして使うことは結局なかった。）

```suggestion
 * \attention In general, you should not close persistent objects such as tables
 *            and columns for performance. If you close persistent objects, you need
 *            to re-open them when they are needed again. It's inefficient.
```